manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       dnsPolicy: "Default"
       containers:
       - name: node-driver-registrar
-        image: quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
+        image: quay.io/k8scsi/csi-node-driver-registrar:v2.1.0
```

Comment:
v2.0.1? https://github.com/kubernetes-csi/node-driver-registrar/releases

Reply:
Fixed. Thanks for catching this.